lib/fx/adapters/postgres/functions.rb

```diff
@@ -32,7 +35,8 @@ def initialize(connection)
           @connection = connection
         end
 
-        # All of the functions that this connection has defined.
+        # All of the functions that this connection has defined. Functions with
```

Comment:
This comment is currently a lie, anyway: functions are _not_ grouped. The initial plan was for Function objects to contain all the different signatures, but that was making things too complex for no reason. I'll remove this comment, I don't believe updating it adds anything as the original comment still completely describes its behavior.